src/zapytania23.1.sql

```diff
@@ -0,0 +1,118 @@
+-- Tworzy tabelę pracownik(imie, nazwisko, wyplata, data urodzenia, stanowisko). W tabeli mogą być dodatkowe kolumny, które uznasz za niezbędne.
+
+CREATE TABLE employees (
+	id INT primary key auto_increment,
+    first_name varchar(30),
+    last_name varchar(30),
+    payment decimal(8,2),
+    birth_date date,
+	position varchar(30)
+);
+
+-- Wstawia do tabeli co najmniej 6 pracowników
+
+INSERT INTO employees 
+	(first_name, last_name, payment, birth_date, position)
+VALUES 
+	('Maria', 'Kowalska', 3200, '2000-10-12', 'sekretarka'),
+    ('Jan' ,'Kowalski' ,5000 ,'1998-07-12' ,'analityk'),
+    ('Marcin' ,'Malinowski' ,5200 ,'1994-05-23' ,'analityk'),
+    ('Hanna' ,'Puchalska' ,3800 ,'1994-06-10' ,'kadrowa'),
+    ('Anna' ,'Karp' ,4200 ,'1986-07-15' ,'księgowa'),
+    ('Alina' ,'Nowak' ,4100 ,'1989-01-12' ,'księgowa');
+    
+-- Pobiera wszystkich pracowników i wyświetla ich w kolejności alfabetycznej po nazwisku
+
+SELECT * FROM employees order by last_name ASC;
+
+-- Pobiera pracowników na wybranym stanowisku
+
+SELECT * FROM employees where position LIKE 'analityk';
+
+-- Pobiera pracowników, którzy mają co najmniej 30 lat
+
+select * from employees where year(curdate())-year(birth_date)>=30;
+
+-- Zwiększa wypłatę pracowników na wybranym stanowisku o 10%
+
+-- NIE DZIAŁA - nie wiem czemu
+update employees set payment = ((select employees.payment where position = 'kadrowa') * 1.1) ; 
+
+-- Pobiera najmłodszego pracowników (uwzględnij przypadek, że może być kilku urodzonych tego samego dnia)
+
+select * from employees where birth_date =(select max(birth_date) from employees);
+
+-- Usuwa tabelę pracownik
+
+drop table employees;
+
+-- Tworzy tabelę stanowisko (nazwa stanowiska, opis, wypłata na danym stanowisku)
+CREATE TABLE title (
+	id INT primary key auto_increment,
+    title_name varchar(30),
+    desc_title varchar(30),
```

Comment:
trzymałbym się jednej konwencji, albo title dodawane na początku, albo na końcu, a najlepiej podobnie jak w przypadku klas w Javie - skoro mamy tabelę title, to nie ma sensu powtarzać `title` w nazwach kolumn